Update BigQuery language (#7851)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/connect-data-platform/connect-bigquery.md

### BigQuery OAuth
**Available in:** Development environments, Enterprise-tier plans only

> The OAuth auth method permits <Constant name="cloud" /> to run development queries on behalf of a BigQuery user without storing the BigQuery service account keyfile in <Constant name="cloud" />. However, the JSON must still be provided, or fields must be manually filled out to complete the configuration in dbt Cloud. For more information on the initial configuration of a BigQuery OAuth connection in <Constant name="cloud" />>, please see [the docs on setting up BigQuery OAuth](/docs/cloud/manage-access/set-up-bigquery-oauth).

Reviewer: I would update this part to account for WIF (deployment auth): "run queries on behalf of a BigQuery user or workload"

> The OAuth auth method permits <Constant name="cloud" /> to run development queries on behalf of a BigQuery user without storing the BigQuery service account keyfile in <Constant name="cloud" />. However, the JSON must still be provided, or fields must be manually filled out to complete the configuration in dbt Cloud. For more information on the initial configuration of a BigQuery OAuth connection in <Constant name="cloud" />>, please see [the docs on setting up BigQuery OAuth](/docs/cloud/manage-access/set-up-bigquery-oauth).

Author: ```suggestion
The OAuth auth method permits <Constant name="cloud" /> to run queries on behalf of a BigQuery user or workload without storing the BigQuery service account keyfile in <Constant name="cloud" />. However, the JSON must still be provided, or fields must be manually filled out to complete the configuration in dbt Cloud. For more information on the initial configuration of a BigQuery OAuth connection in <Constant name="cloud" />>, please see [the docs on setting up BigQuery OAuth](/docs/cloud/manage-access/set-up-bigquery-oauth).
```

> The OAuth auth method permits <Constant name="cloud" /> to run queries on behalf of a BigQuery user or workload without storing the BigQuery service account keyfile in <Constant name="cloud" />. However, the JSON must still be provided, or fields must be manually filled out to complete the configuration in dbt Cloud. For more information on the initial configuration of a BigQuery OAuth connection in <Constant name="cloud" />>, please see [the docs on setting up BigQuery OAuth](/docs/cloud/manage-access/set-up-bigquery-oauth).

Reviewer 2: ```suggestion
The OAuth auth method permits <Constant name="cloud" /> to run queries on behalf of a BigQuery user or workload without storing the BigQuery service account keyfile in <Constant name="cloud" />. However, the JSON must still be provided, or fields must be manually filled out to complete the configuration in dbt Cloud. Those values do not have be real in order for this bypass to work. For more information on the initial configuration of a BigQuery OAuth connection in <Constant name="cloud" />>, please see [the docs on setting up BigQuery OAuth](/docs/cloud/manage-access/set-up-bigquery-oauth).
```

Reviewer 2: I don't know how to say basically you can just type in `blah, blah` and this will work. Or maybe we give a value like `N/A` so everyoen does the same thing?

> The OAuth auth method permits <Constant name="cloud" /> to run queries on behalf of a BigQuery user or workload without storing the BigQuery service account keyfile in <Constant name="cloud" />. However, the JSON must still be provided, or fields must be manually filled out to complete the configuration in dbt Cloud. Those values do not have be real in order for this bypass to work. For more information on the initial configuration of a BigQuery OAuth connection in <Constant name="cloud" />>, please see [the docs on setting up BigQuery OAuth](/docs/cloud/manage-access/set-up-bigquery-oauth).

Author: ```suggestion
The OAuth auth method permits <Constant name="cloud" /> to run queries on behalf of a BigQuery user or workload without storing the BigQuery service account keyfile in <Constant name="cloud" />. However, the JSON must still be provided, or fields must be manually filled out to complete the configuration in dbt Cloud. Those values do not have to be real for this bypass to work (for example, they can be `N/A`). For more information on the initial configuration of a BigQuery OAuth connection in <Constant name="cloud" />>, please see [the docs on setting up BigQuery OAuth](/docs/cloud/manage-access/set-up-bigquery-oauth).
```
